### DISCUSSION TITLE
Is there any plan to create a Discord server?

### DISCUSSION THREAD

**Discussion by g-libardi** [piccolo-orm/piccolo#1066] [General]
I came across a discussion from 2021 about this, but it seems the idea didn't move forward. If that's the case, do you plan on pursuing it? Among the current Python ORMs, I believe Piccolo offers one of the best developer experiences. This could significantly boost its popularity by engaging more with the community.

**Comment by dantownsend** [comment]
Hey, sorry for the slow reply.

You're right, we should have one. I like GitHub Discussions because we can easily create issues from discussions. But it's not a great place for building community.

Here's a link to a Discord server: https://discord.com/invite/JbXBgPM2GY

I'm not a Discord expert. My plan for now is to at the very least post about new releases there, and try and answer questions.

Thanks
